configs/edison/config.20161006bugfix.alpha8.A_WCYCL1850S.ne30_oEC_ICG.edison

```diff
@@ -131,6 +131,7 @@ polarPlot = False
 [streamfunctionMOC]
 ## options related to plotting the streamfunction of the meridional overturning 
 ## circulation (MOC)
+maxChunkSize = 1000
```

Comment:
We may need to do something like this (http://xarray.pydata.org/en/stable/dask.html):
> Save intermediate results to disk as a netCDF files (using to_netcdf()) and then load them again with open_dataset() for further computations. For example, if subtracting temporal mean from a dataset, save the temporal mean to disk before subtracting. Again, in theory, dask should be able to do the computation in a streaming fashion, but in practice this is a fail case for the dask scheduler, because it tries to keep every chunk of an array that it computes in memory. (See dask issue #874)

This is a dask bug that I wasn't aware of but it sounds like the solution is simple-- we write out the climatology and then read it in.  The file size is approx 1-2 gb so perhaps this won't be a huge problem.  It will also provide a way to cache.  Having a generalized multi-scale approach to building out the climatologies, which necessarily would be having the right file organizational structure, is probably what we need as a long-term solution as we discussed in the meeting yesterday.